src/components/timeline/TimelineViewControls.svelte

```diff
@@ -16,7 +16,7 @@
   import TimelineViewDirectiveControls from './TimelineViewDirectiveControls.svelte';
 
   export let maxTimeRange: TimeRange = { end: 0, start: 0 };
-  export let minZoomMS = 100; // Min zoom of one minute
+  export let minZoomMS = 10;
```

Comment:
I think this disabling bit no longer works? Can zoom in infinitely and then eventually see nothing-ish. Do we want to just get rid of this minZoomMS or enforce a min?

Reply:
I'm thinking that there's no need to enforce a minimum. I'm not able to come up with a case where enforcing a zoom minimum helps much.